pkg/promunifi/uap.go

```diff
@@ -195,8 +197,8 @@ func (u *promUnifi) exportUAP(r report, d *unifi.UAP) {
 
 func (u *promUnifi) exportUAPstats(r report, labels []string, ap *unifi.Ap) {
 	//	labelA := append([]string{"all"}, labels[2:]...)
-	labelU := append([]string{"user"}, labels[6:]...)
-	labelG := append([]string{"guest"}, labels[6:]...)
+	labelU := append([]string{"user"}, labels[1:]...)
+	labelG := append([]string{"guest"}, labels[1:]...)
```

Comment:
I'd just write these as `labelG := []string{"guest", labels[1], labels[2]}` - this looks cleaner once you get to only a couple items in the list.

Reply:
probably the same for the following two.

Reply:
Yea, probably safer.

Reply:
Fixed up.